### PR TITLE
Request that the prescriptions show additional information

### PR DESCRIPTION
**Pull request by juggernautsei**
Dr. Haggerty in Knoxville TN requested that this information be added to the prescriptions like the medications display.  His office manager says that the state requires this information to be on screen.

below added by bradymiller:
Up For Grabs demo for this PR is here: https://www.open-emr.org/wiki/index.php/Development_Demo#Epsilon_-_Up_For_Grabs_Demo

### CODE REVIEW

**Comment by juggernautsei** [comment]
@robertdown @jacobzh  @bradymiller 
Please review.

File: library/classes/Prescription.class.php [hunk 1]
```diff
@@ -667,7 +667,14 @@ function get_active()
     {
         return $this->active;
     }
+    function get_dose_size_display()
```
Comment (bradymiller) [inline library/classes/Prescription.class.php]: I am not seeing this function being used below??

File: templates/prescription/general_fragment.html [hunk 2]
```diff
@@ -1,24 +1,29 @@
...
-  {if $prescription->get_active() > 0}
-	<tr class='text'>
-		<td>{$prescription->drug|text}</td>
-		<td>{$prescription->get_dosage_display()|text}</td>
-	</tr>
-  {/if}
-	{/foreach}
+    {if empty($prescriptions)}
+    <tr class='text'>
+        <td>&nbsp;&nbsp;{xl t='None'|escape:'html'}</td>
```
Comment (bradymiller) [inline templates/prescription/general_fragment.html]: revert above line (also note below where you've changed prior code that should also be reverted)

File: templates/prescription/general_fragment.html [hunk 3]
```diff
@@ -1,24 +1,29 @@
...
+    </tr>
+    {/if}
+    {foreach from=$prescriptions item=prescription}
+    {if $prescription->get_active() > 0}
+    <tr class='text'>
+        <td>{$prescription->drug|escape:'html'}&nbsp;</td>
```
Comment (bradymiller) [inline templates/prescription/general_fragment.html]: revert above line

File: templates/prescription/general_fragment.html [hunk 4]
```diff
@@ -1,24 +1,29 @@
...
+        <td>{$prescription->get_size()|escape:'html'}{$prescription->get_unit_display()|escape:'html'}&nbsp;
+            {$prescription->get_dosage_display()|escape:'html'}</td>
```
Comment (bradymiller) [inline templates/prescription/general_fragment.html]: revert above line

File: templates/prescription/general_fragment.html [hunk 5]
```diff
@@ -1,24 +1,29 @@
...
+    {if empty($prescriptions)}
+    <tr class='text'>
+        <td>&nbsp;&nbsp;{xl t='None'|escape:'html'}</td>
+    </tr>
+    {/if}
+    {foreach from=$prescriptions item=prescription}
+    {if $prescription->get_active() > 0}
+    <tr class='text'>
+        <td>{$prescription->drug|escape:'html'}&nbsp;</td>
+        <td>{$prescription->get_size()|escape:'html'}{$prescription->get_unit_display()|escape:'html'}&nbsp;
```
Comment (bradymiller) [inline templates/prescription/general_fragment.html]: above line should be:
```
<td>{$prescription->get_size()|text}{$prescription->get_unit_display()|text}&nbsp;
```

**Comment by bradymiller** [comment]
review complete

**Comment by bradymiller** [comment]
Up For Grabs demo for this PR is here: https://www.open-emr.org/wiki/index.php/Development_Demo#Epsilon_-_Up_For_Grabs_Demo

**Comment by bradymiller** [comment]
hi @juggernautsei , Code looks good and testing well. Bringing this into codebase. Thanks for the code improvement! -brady

**Pull request closed without merging**